### DISCUSSION TITLE
how to restore my friend's account

### DISCUSSION THREAD

**Discussion by liaoyugen** [community/community#31557] [New to GitHub] [answered]
Hello github admin,

one of my friend's account "jiajue7451" was suspended recently
since we thought  he was loyal and junior user in github, we don't know why his account  was blocked

we have committed github support tickets for about 5 days, but got no response yet
So could some one help to slove theseissue?

if he violated some rules, please let us know,we will cooperate with admin to correct them. 
Thanks a lot, any help will be highly appreciated！

**Comment by toufiqul-hasan** [comment]
Maybe some incorrect and automatic spam identification? You should contact them: [Contact us - GitHub Support](https://support.github.com/contact?tags=dotcom-footer)

**Reply by liaoyugen** [reply]
thanks you  for the answering!
i have already committed 2 tickets for the account issue, but got no response. 
i think i have to wait for the respone from github support!

**Comment by melvincwng** [comment, marked as answer]
@liaoyugen, unfortunately for account related issues, members of the GitHub Community won't be able to help or comment much.

The only way is to wait a little longer for GitHub Support to contact you.

They could be swamped with multiple requests or tickets right now, and hence the delay in responding to you or your friend.

Hope this clarifies 😊
